Frontend/src/App.js

```diff
@@ -50,7 +53,7 @@ function App() {
 
   const handleCopyClick = () => {
     setContentToCopy(ltxcbRef.current.innerText);
-    navigator.clipboard.writeText(contentToCopy)
+    navigator.clipboard.writeText(`$${contentToCopy}$`)
```

Comment:
That would work ig but we need to show dollar sign in output as well, not just in clipboard.
it's like user will see `1234/sigma` but in clipboard it would be `$1234/sigma$`